### PR TITLE
Test pull request

### PR DESCRIPTION
**Pull request by jaylee0170**
I am making a pull request

### CODE REVIEW

**Comment by junlee-db** [comment]
Hello.

File: lab1/HelloWorld.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 public class HelloWorld {
     public static void main(String[] args) {
-        System.out.println("Helloo world!");
+        System.out.println("Hellooo world!");
```
Comment (junlee-db) [inline lab1/HelloWorld.java]: I think this o is too fat. Reduce it. :hehe:

Comment (jaylee0170) [reply lab1/HelloWorld.java:3]: no.

**Comment by junlee-db** [comment]
LGTM!

**Review by junlee-db — APPROVED** [review]

**Pull request merged**